sass/components/_emoticons.scss

```diff
@@ -181,6 +181,7 @@
     padding: 13px 10px 13px 15px;
     background: $primary-color;
     border: 1px solid $light-gray;
+    border-color: rgba(var(--center-channel-color-rgb),0.2);
```

Comment:
Could you combine these two lines so that we're not saying the border will be grey and then immediately giving it a colour?